lib/app/client.js

```diff
@@ -115,7 +115,7 @@ async function loadAsyncComponents (to, from, next) {
   this._diffQuery = (this._queryChanged ? getQueryDiff(to.query, from.query) : [])
 
   <% if (loading) { %>
-  if (this._pathChanged && this.$loading.start) {
+  if (this._pathChanged && this.$loading.start && !this.$loading.manual) {
```

Comment:
This line should be called before `render` in `client.js`, but `manual` is initialized in `render`, so if in `spa` mode, can this line work ?

Reply:
@clarkdo good catch, I'll look into it.

Reply:
@Atinux @clarkdo so I didn't forget this, and I am happy to report the `loading: false` works flawlessly on SPA mode :)